src/pmdas/linux_zfs/zfs_dmu_tx.c

```diff
@@ -39,10 +39,12 @@ zfs_dmu_tx_refresh(zfs_dmu_tx_t *dmu_tx)
             mname = strtok(line, delim);
             mval  = strtok(NULL, delim); // not used
             mval  = strtok(NULL, delim);
-            value = strtoull(mval, NULL, 0);
 
-            if (strncmp(mname, "dmu_tx_", 7) != 0)
```

Comment:
I think we still need this line...?  What if the file format was (perhaps changed in the future) such that a different string prefix existed for some lines in the file?